freqtrade/freqai/freqai_interface.py

```diff
@@ -579,6 +588,28 @@ def fit_live_predictions(self, dk: FreqaiDataKitchen, pair: str) -> None:
 
         return
 
+    def inference_timer(self, do='start'):
```

Comment:
I think this would be something that might be moved someplace else in the future.

"slow" strategies can have the same effect, with the same risk.
That said - i'm not sure if we really need this - setting `ignore_buying_expired_candle_after` can already do this automatically - automatically ignoring "late" entries.